binding/HarfBuzzSharp.NativeAssets.Linux/HarfBuzzSharp.NativeAssets.Linux.csproj

```diff
@@ -10,11 +10,13 @@
     <PackageFile Include="..\..\output\native\linux\x86\libHarfBuzzSharp*" PackagePath="runtimes\linux-x86\native\%(Filename)%(Extension)" />
```

Comment:
Not related to your chnages, but I noticed that there are 2 .so files in each folder because of this wildcard. Could you also replace these items to rather be `libHarfBuzzSharp*.so` intead of just `libHarfBuzzSharp*`.